source/by-role/developer/common/experience-builder.rst

```diff
@@ -1575,6 +1575,38 @@ Angular, Vue and Other JS Applications
 The easiest way to integrate XB with your JS application is by putting attributes on each HTML element that
 represents a model, field or item of a CrafterCMS content type and then invoking XB initializer.
 
+To initialize XB, you need to invoke the ``initExperienceBuilder`` function. This function receives a single argument which gets passed down to the :ref:`ExperienceBuilder component <ExperienceBuilder>`. See argument details on the ExperienceBuilder component section.
+At a minimum, you need to supply the ``isAuthoring`` boolean flag, together with either a ``ContentInstance`` or the ``path`` for the main model to initialize XB with.
+For example, a simple application that uses the UMD bundle for XB, would look something like this:
+
+.. code-block:: js
```

Comment:
Is there HTML highlight? That should highlight correctly both the tags and the JS inside script.